decision_service/include/ds_ranking_response.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace decision_service {
+
+	//Ranking response returned by the decision service
+	class ranking_response {
+
+	public:
+		ranking_response();
+		ranking_response(const std::string& uuid, const std::vector<std::pair<int, float>>& ranking);
```

Comment:
When we discussed the name of this previously, I thought Rajan had a more neutral name thank 'ranking'?

Reply:
I remember choose_rank() method name.  I don't recall response class name ... Maybe @yannstad ?

---
In reply to: [187118946](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187118946) [](ancestors = 187118946)

Reply:
I don't recall either.
If we rename the class `ranking_response` in `ranking`, i suggest to also rename the `get_ranking()` and `set_ranking()` methods from the `ranking_response`.

Reply:
choose_rank() was what I was thinking of.   Why aren't we using that in ds_driver.h?